packages/instantsearch.js/src/middlewares/__tests__/createInsightsMiddleware.ts

```diff
@@ -185,6 +185,49 @@ describe('insights', () => {
       expect((window as any).aa).toEqual(expect.any(Function));
     });
 
+    it('loads script, even when globals are set up by a different instance', () => {
```

Comment:
It's clear to us now, but in the future this test doesn't tell me much about when it can happen.

Could we have a test in the RISH packages, using strict mode instead?